.github/workflows/fuzzing.yml

```diff
@@ -0,0 +1,41 @@
+name: fuzzing
+
+on: [push]
+
+jobs:
+  fuzzing:
+    # We want to run on external PRs, but not on our own internal PRs
+    # as they'll be run by the push to the branch.
+    if: ( github.event_name == 'push' ||
+        github.event.pull_request.head.repo.full_name != github.repository ) &&
+        ! endsWith(github.ref, '-notest')
```

Comment:
If I understand the logic correctly and considering `on: [push]` above:

1. `event_name` always will be `push`.
2. `pull_request` will never be present.

And there is no filtration by the repository owner/name for a push event. It contradicts with our wish (see the discussion in the PR). We should not run the job for a push inside a fork.

Please, correct me if I'm wrong: I'm not much experienced around GitHub Actions.

Reply:
And I don't see filtering by a branch name (master/2.*/1.10).

Reply:
According to my [comment](https://github.com/tarantool/tarantool/pull/5841#issuecomment-782697080) below: we should skip the job on 2.7/2.6/1.10 too. Run it only on push to `master` in tarantool/tarantool repository.

Reply:
Done.

```
--- a/.github/workflows/fuzzing.yml
+++ b/.github/workflows/fuzzing.yml
@@ -2,6 +2,8 @@ name: fuzzing
 
 on:
   push:
+    branches:
+      - 'master'
     paths:
       - '.github/workflows/**'
       - 'src/**'
```